src/test/java/org/folio/readingroom/controller/PatronPermissionControllerTest.java

```diff
@@ -0,0 +1,90 @@
+package org.folio.readingroom.controller;
+
+import static org.folio.readingroom.utils.HelperUtils.READING_ROOM_ID;
+import static org.folio.readingroom.utils.HelperUtils.SERVICE_POINT_ID1;
+import static org.folio.readingroom.utils.HelperUtils.SERVICE_POINT_ID2;
+import static org.folio.readingroom.utils.HelperUtils.SERVICE_POINT_NAME1;
+import static org.folio.readingroom.utils.HelperUtils.SERVICE_POINT_NAME2;
+import static org.folio.readingroom.utils.HelperUtils.createPatronPermission;
+import static org.folio.readingroom.utils.HelperUtils.createReadingRoom;
+import static org.folio.readingroom.utils.HelperUtils.createServicePoint;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.put;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+import java.util.Set;
+import java.util.UUID;
+import org.folio.readingroom.client.feign.UsersClient;
+import org.folio.readingroom.domain.dto.PatronPermission;
+import org.folio.readingroom.domain.dto.ReadingRoom;
+import org.folio.readingroom.repository.PatonPermissionsRepository;
+import org.folio.readingroom.repository.ReadingRoomRepository;
+import org.folio.spring.service.SystemUserScopedExecutionService;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.MediaType;
+
+class PatronPermissionControllerTest extends BaseIT {
+
```

Comment:
I think the below fields are not used anywhere in this class. If it is not used, we can remove all these fields.